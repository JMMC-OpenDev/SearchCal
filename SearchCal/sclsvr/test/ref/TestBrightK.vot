version https://git-lfs.github.com/spec/v1
oid sha256:9107816057e03903a52a6f4ae6cb2039b70479d42fa0411d9f98790b4ac27506
size 325312
