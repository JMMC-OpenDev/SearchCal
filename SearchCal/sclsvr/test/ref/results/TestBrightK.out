version https://git-lfs.github.com/spec/v1
oid sha256:cacdc4b9c21a07fe90528bb4e4e30fedf2c5b173fae1024c7ca09339c36a2f6f
size 1010967
